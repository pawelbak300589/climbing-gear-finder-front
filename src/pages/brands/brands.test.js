import React from 'react';
import { shallow } from "enzyme";

import BrandsPage from "./brands.component";

it('should render BrandsPage component', () => {
    const mockMatch = {
        path: '/brands'
    };

    expect(shallow(<BrandsPage match={mockMatch} />)).toMatchSnapshot();
});