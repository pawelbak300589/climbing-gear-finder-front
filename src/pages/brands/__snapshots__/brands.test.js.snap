// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render BrandsPage component 1`] = `ShallowWrapper {}`;
